testprojects/src/python/interpreter_selection/python_3_binary/lib.py

```diff
@@ -0,0 +1,16 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import sys
+
+
+# A simple example to include in a python 3 binary target
+
+def say_something():
+  print('I am a python 3 library method.')
+  assert 1/2 == 0.5
+  return 1/2  # for testing the `./pants run` task
```

Comment:
would be good to add a comment here noting that the `assert 1/2 == 0.5` bit is a python3 specific behavior check for casual onlookers - or else use something more explicitly python3-y in this example, like `asyncio`.